ethereum/contracts/TablelandRigs.sol

```diff
@@ -425,7 +442,17 @@ contract TablelandRigs is
         if (ownerOf(tokenId) != _msgSenderERC721A())
             revert ITablelandRigPilots.Unauthorized();
 
-        _pilots.pilotRig(_msgSenderERC721A(), tokenId, pilotAddr, pilotId);
+        // If the supplied pilot address is `0x0`, then assume a trainer pilot
+        // (note: `pilotId` has no impact here). Otherwise, proceed with a
+        // custom pilot. The overloaded methods direct changes accordingly.
+        pilotAddr == address(0)
```

Comment:
This is the only logic update in the Rigs contract, so there aren't any fn signature changes. You can now pass `0x0` as contract value, which will assume you're trying to pilot your Rig with a "stock" trainer pilot. 

In this situation, only the passed `pilotAddr` matters; `pilotId` has no impact within the Rigs contract itself but is hardcoded in the `TablelandRigPilots` contract to a value of `2`.